src/main/java/org/folio/consortia/service/impl/CapabilitiesUserServiceImpl.java

```diff
@@ -40,35 +38,25 @@ public class CapabilitiesUserService implements PermissionUserService {
   private final CapabilitySetsClient capabilitySetsClient;
   private final UserCapabilitiesClient userCapabilitiesClient;
   private final UserCapabilitySetsClient userCapabilitySetsClient;
-  private final UserPermissionsClient userPermissionsClient;
+  private final UserRolesClient userRolesClient;
   private final ObjectMapper objectMapper;
 
   @Override
-  public Optional<PermissionUser> getByUserId(String userId) {
-    return Optional.of(userPermissionsClient.getPermissionsForUser(userId, false));
+  public void createWithPermissionSetsFromFile(String userId, String permissionSetsFilePath) {
+    var perms = readAndValidatePermissionSets(permissionSetsFilePath);
+    log.info("createWithPermissionSetsFromFile:: Assigning permission sets: [{}] to user: '{}'", perms, userId);
+    assignPermissionSets(userId, perms);
   }
 
   @Override
-  public PermissionUser createWithEmptyPermissions(String userId) {
-    throw new UnsupportedOperationException("User cannot be assigned with empty set of capabilities");
-  }
-
-  @Override
-  public PermissionUser createWithPermissionSetsFromFile(String userId, String permissionSetsFilePath) {
-    var perms = readAndValidatePermissions(permissionSetsFilePath);
-    var permissionUser = PermissionUser.of(UUID.randomUUID().toString(), userId, perms);
-    log.info("Creating permissionUser {}.", permissionUser);
-    assignPermissionSets(permissionUser.getUserId(), perms);
-    return permissionUser;
-  }
-
-  @Override
-  public void deletePermissionUser(String userId) {
+  public void deleteUserCapabilitiesAndRoles(String userId) {
     userCapabilitiesClient.deleteUserCapabilities(userId);
-    log.info("deleteUserPermissions:: Deleted capabilities with userId={}", userId);
+    userCapabilitySetsClient.deleteUserCapabilitySets(userId);
```

Comment:
need to consider case where userid is not exists not found exception case. example of different cases you can use appopriate one if need
```
} catch (FeignException.NotFound e) {
      log.info("User with userId {} does not exist in schema, going to use new one", userId);
      return new User();
    } catch (FeignException.Forbidden e) {
      throw new ConsortiumClientException(e);
    } catch (FeignException e) {
      throw new IllegalStateException(e);
    }
```

Reply:
Thanks for noticing, now added support to call each even if any of them return not found exception